src/Symfony/Component/VarDumper/Tests/Dumper/CliDumperTest.php

```diff
@@ -230,7 +230,20 @@ public function testJsonCast()
         $var[] = &$v;
         $var[''] = 2;
 
-        if (\PHP_VERSION_ID >= 70200) {
+        if (\PHP_VERSION_ID >= 70400) {
+            $this->assertDumpMatchesFormat(
+                <<<'EOTXT'
+array:4 [
+  0 => & {}
```

Comment:
That's a PHP7.4 bug.
I reported it as https://bugs.php.net/77951
Please revert the test case and let's wait for the issue to be fixed before merging.